docs/marketplace/dev/architecture-overview/202108.0/marketplace-merchant-portal-architecture-overview.md

```diff
@@ -0,0 +1,33 @@
+---
+title: Merchant Portal Architecture overview
+description:
+template: concept-topic-template
+---
+This document depicts how a Spryker Marketplace MerchantPortal is designed in general. 
+
+The following diagrams outlines the relation between Zed, MerchantPortal, Backoffice and DB. Learn more about [Marketplace MerchantPortal Core feature here](/docs/marketplace/dev/feature-walkthroughs/202108.0/marketplace-merchant-portal-core-feature-walkthrough/marketplace-merchant-portal-core-feature-walkthrough.html).
+
+![MerchantPortal Architecture overview](https://confluence-connect.gliffy.net/embed/image/4b06167a-3c9a-483c-8b57-32544b211fc5.png?utm_medium=live&utm_source=custom)
+
+### Zed and MerchantPortal
+Zed is an application layer at Spryker (next to Yves, Glue, Client, Service, and Shared). 
+
+This layer serves as a base for quite some of backend-oriented applications such as MerchantPortal, Backoffice, Gateway, Console (DataImport, Pub&Sync). That means that MerchantPortal shares the codebase with these applications and all the internal Zed infrastructure is available withing MerchantPortal runtime. This allows faster development and easier customizations of your Spryker Marketplace project.  
+
+### Security
+While addressing different concerns, both MerchantPortal and Backoffice has direct access the main database where all the application transactions are stored.
```

Comment:
You need to turn it over from the problem statement to the actual solution,
how MP is addressing the security concern. 

If there are risks, that should be addressed by MP,
and if they are addressed they are not risks anymore.

Reply:
While we addressing these risks, they still remain valid due architecture design we took.
We need to warn our customers to make the to be aware and cautious on this important topic .